pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -44,11 +45,17 @@ def __init__(self, trainer: "pl.Trainer", log_gpu_memory: Optional[str] = None)
     def on_trainer_init(
         self,
         logger: Union[bool, LightningLoggerBase, Iterable[LightningLoggerBase]],
-        flush_logs_every_n_steps: int,
+        flush_logs_every_n_steps: Optional[int],
         log_every_n_steps: int,
         move_metrics_to_cpu: bool,
     ) -> None:
         self.configure_logger(logger)
+        if flush_logs_every_n_steps is not None:
+            rank_zero_deprecation(
+                f"Setting `Trainer(flush_logs_every_n_steps={flush_logs_every_n_steps})` is deprecated in v1.5 "
+                "and will be removed in v1.7. Please configure flushing in the logger instead."
+            )
+        flush_logs_every_n_steps = 100  # original default parameter
```

Comment:
since it's deprecated and not removed entirely, should the logger still consider this value if not provided in the logger explicitly?

Reply:
@edward-io is an indentation issue? idd you mean to apply this if `flush_logs_every_n_steps is None` ?